### PR TITLE
[WWW-82] Update RSS feed to reflect all historical terms of service updates

### PR DESCRIPTION
**Pull request by klijakub**
Add all entries in the Terms of Service CHANGELOG to the /legal.rss feed. Some guidelines:

Create one <item> for each CHANGELOG entry. ✔ Done

Use the GitHub commit link as the <link>. ✔ Done

You can copy the text verbatim as is, but feel free to fix any typos you find. 
✔ Done

Delete the Terms of Service CHANGELOG page. 
✔ Done We removed the whole changelog folder from page dir.

Update the link at the top of https://gruntwork.io/terms to point to WWW-78: Render RSS feed on site.
✔ Done

Jira Item: [WWW-82]

[WWW-82]: https://gruntwork.atlassian.net/browse/WWW-82

### CODE REVIEW

File: _data/legal.yml [hunk 1]
```diff
@@ -3,27 +3,91 @@
   description: Going forward, all updates to the Gruntwork terms of service, privacy policy, open source policy, and other policies relating to data processing will be posted here on our our RSS feed
   category: privacy-policy
   link: https://gruntwork.io/
```
Comment (oredavids) [inline _data/legal.yml:5]: Please update this link to the RSS feed link.
```suggestion
  link: https://gruntwork.io/legal.rss
```

**Comment by klijakub** [comment]
Hello @oredavids ;)
Thank you for your review! 
I've added missing guid numbers but I had to reorder entries and change the guid numbers to keep the correct order in legal.yml file. Hope that's not a problem.

**Comment by oredavids** [comment]
@klijakub no problem at all. That needed to happen.

**Comment by josh-padnick** [comment]
Thanks @klijakub and @oredavids for your contributions. This looks ready to merge! Approving now.

**Pull request merged**